pysat/utils/time.py

```diff
@@ -269,57 +269,40 @@ def create_datetime_index(year=None, month=None, day=None, uts=None):
 
     """
 
-    # We need a timeseries index for storing satellite data in pandas, but
```

Comment:
Closest place to put the comment. Not your text, but the description for `uts` is wonky. 
```
Array of UT seconds as np.float64 .....
```